### DISCUSSION TITLE
Elevated: no succes, reinstall of the integration messed entity names

### DISCUSSION THREAD

**Discussion by MEKadan** [wlcrs/huawei_solar#600] [Setting up the integration]
In the reaDme: 

"Q: I didn't check 'Advanced: Elevate permissions' during the initial setup of this integration and changed my mind. How do I change this?
A: Uninstall and reinstall this integration. You will not lose the history of your sensors."

I did. It does not accept my initials, no matter what I do.
Had to go back and try to install with non-elevated permissions. OK, but e.g. the sensor.active_power was renamed to sensor.inverter_active_power and sensor.daily_yield to sensor.inverter_daily_yieLd.

I lost ALL MY HISTORY. 

And the question is, HOW DO I GET IT BACK?

**Comment by jires** [comment]
> I did. It does not accept my initials, no matter what I do.

Initials?? You need to type Installer credentials, the same as you using in FusionSolar app while commissioning. (when you using sdongle to connect, it can be still factory password (00000a).



> I lost ALL MY HISTORY.

Just rename entities manually to old ones, so your history will came back.

**Comment by MEKadan** [comment]
Phew... Thanks, @jires  !

1. The entities are now renamed and the data is back, Scared the h*ll out of me,  and did not remember this simple "trick"
2. Yes, I meant credentials, installer credentials to be exact. I can log in with Fusion Solar, but HA integration does not let me in with same credentials. However, it has been almost 2 years after I las touched these, so cannot be sure, but I think I'm not using Sdongle, even though I have it. I think I'm connecting to the unit's own wifi.

The connection is pretty much identical to one presented in this post: https://github.com/wlcrs/huawei_solar/discussions/508

...the only difference being, that I do have an extra wireless router between my home network and Huawei. The connection has been OK, after I got it working, which was not an easy task. It is an TP-Link WR802N with openwrt firmware.  If you are interested in this story, it begins from 1887 from this very long thread https://community.home-assistant.io/t/integration-solar-inverter-huawei-2000l/132350/1887

However, the point probably is that Huawei gets IP 192.168.201, just like in the first link, port also being the same, 6607. I suppose this means it is the internal wifi of Huawei.

Why I told all this, is that maybe you or someone can provide some hints on how to check that Huawei internal wifi credentials are the same  than I rhink they are. Or is it possible that there are some default credentials or like still active? What could those be?

**Comment by MEKadan** [comment]
I checked once again: My Installer credentials are OK in Fusion Solar app, but HA integration does not accept them.
